src/one_dragon_qt/widgets/download_card/launcher_download_card.py

```
@@ -55,6 +60,76 @@ def run(self) -> None:
+        self.check_result: RuntimeUpdateCheckResult | None = None
+        self.source_mode: str = RuntimeUpdateSourceEnum.AUTO.value.value
+        self.progress_signal: dict[str, str | None] = {"signal": None}
+
+    def run(self) -> None:
+        if self.check_result is None:
+            self.finished.emit(False, "未找到可用更新", None)
+            return
+
+        try:
+            prepared = self._download_preferred_update(self.check_result)
+            self.finished.emit(True, "准备更新完成，即将重启应用", prepared)
```

Ruff (0.15.5)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)